specifyweb/frontend/js_src/lib/utils/uniquifyName.ts

```diff
@@ -55,7 +54,13 @@ export function getUniqueName(
     newIndex === 1 && length === 0
       ? strippedName
       : `${strippedName}${uniquePart}`;
+
   return newName.length > maxLength
-    ? getUniqueName(name.slice(0, -1 * uniquePart.length), usedNames, maxLength)
+    ? getUniqueName(
+        name.slice(0, -1 * uniquePart.length),
+        usedNames,
+        maxLength,
+        type
```

Comment:
A minor bug - I forgot to pass `type` in here before
Fixed, and added a test case